Source/WebCore/Modules/mediasource/MediaSource.h

```diff
@@ -38,9 +38,12 @@
 #include "HTMLMediaElement.h"
 #include "MediaSourcePrivateClient.h"
 #include "URLRegistry.h"
+#include <optional>
 #include <wtf/LoggerHelper.h>
+#include <wtf/NativePromise.h>
 #include <wtf/RefCounted.h>
 #include <wtf/UniqueRef.h>
+#include <wtf/Vector.h>
```

Comment:
`optional` and `Vector.h` don't appear to be used in the header. Can these be removed?

Reply:
they were used, I added them because compilation broke elsewhere following unified build changes.
We have `Vector<PlatformTimeRanges> activeRanges()` method .
and we have `std::optional<SeekTarget> m_pendingSeekTarget;` class member